### PR TITLE
chapter13

### PR DESCRIPTION
**Pull request by meimei0111**
ready

### CODE REVIEW

**Comment by meimei0111** [comment]
ready

File: Gemfile [hunk 1]
```diff
@@ -42,3 +45,4 @@ group :test do
 end
 
 gem "tzinfo-data", platforms: [:mingw, :mswin, :x64_mingw, :jruby]
+s
```
Comment (manhnt-0960) [inline Gemfile]: delete "s"

File: app/controllers/static_pages_controller.rb [hunk 2]
```diff
@@ -1,7 +1,14 @@
 class StaticPagesController < ApplicationController
-  def home; end
+	def home
```
Comment (manhnt-0960) [inline app/controllers/static_pages_controller.rb:2]: pls check indent

File: app/views/static_pages/home.html.erb [hunk 3]
```diff
@@ -1,12 +1,30 @@
...
+  <div class="center jumbotron">
+    <% provide :title, t(".home_title") %> 
+    <h1><%= t ".heading" %></h1>
 
-  <p>
-  	<%= t ".description_title" %>
-  	<%= link_to t("ruby_on_rails_tut"), Settings.links.rails_tutorial.home_page %> 
-    <%= t ".description_detail" %>
-  </p>
+		<p>
```
Comment (manhnt-0960) [inline app/views/static_pages/home.html.erb]: indent

**Comment by meimei0111** [comment]
ready

**Comment by ngocvt-0484** [comment]
merged

**Pull request merged**